.github/workflows/build.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ 11 ]
```

Comment:
Perhaps we need both?

Reply:
Either we could kindly ask the CEQ team to annotate their tests with supported Java versions or will need some external config.